msal/src/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -68,6 +67,11 @@ internal AuthenticationResult(MsalAccessTokenCacheItem msalAccessTokenCacheItem,
         /// </summary>
         public virtual string AccessToken => _msalAccessTokenCacheItem.Secret;
 
+        /// <summary>
+        /// 
+        /// </summary>
```

Comment:
you can probably copy/paste the summaries from ADAL counterparts

Reply:
What about: 
In case when Azure AD has an outage, to be more resilient. it can return tokens with an expiration time, and also with an extended expiration time. 
The tokens are then automatically refreshed by MSAL when the time is more than the expiration time, except when ExtendedLifeTimeEnabled is true and the time is less than the extended expiration time. This goes in pair with Web APIs middleware which, when this extended life time is enabled, can accept slightly expired tokens. Client applications accept extended life time tokens only if the ExtendedLifeTimeEnabled Boolean is set to true on ClientApplicationBase

Reply:
fixed